universum/modules/output/html_output.py

```diff
@@ -145,3 +141,10 @@ def _build_html_head():
         head.append('<meta content="utf-8" http-equiv="encoding">')
         head.append(f"<style>{css_rules}</style>")
         return "".join(head)
+
+    @staticmethod
+    def _set_text_style(text, color, bold=False):
+        style = f"color:{color};"
+        if bold:
+            style += "font-weight:bold"
+        return f'<span style="{style}">{text}</span>'
```

Comment:
Please use css class instead of inline style attribute.

Reply:
Done.